src/controllers/authController.js

```diff
@@ -158,9 +158,47 @@ const forgotPassword = async (req, res) => {
   res.status(StatusCodes.OK).json({ message: 'Password reset email sent' });
 };
 
+//reset Password endpoint
+const reset_password = async (req, res) => {
```

Comment:
For consistency, we've been using `camelCase` throughout the app for function names, so let's stick to that here instead of `snake_case`

```suggestion
const resetPassword = async (req, res) => {
```

Reply:
@akosasante sure! done